.kuzzlerc.sample

```diff
@@ -376,8 +376,14 @@
         // value of Access-Control-Allow-Origin header to answer the upgrade request
         "origins": "*:*"
       },
+      // * enabled:
```

Comment:
nitpicking: to stay coherent with other settings above in the file, comments should be placed "inside" the object:
```
       "websocket": { 
           // <!-- Comments here -->
           enabled: true,
           heartbeat: 60000
         }
```